source/documentation/other-topics/aws-rds-migration.md

```diff
@@ -1,21 +1,17 @@
 ### Migrating an RDS instance
 
-This guide covers different ways of migrating RDS instances from one AWS account to another:
-
- - AWS Database Migration Service (DMS)   
- - Pure `pg_dump` & `psql`     
+This guide covers one way of migrating an RDS instance from one AWS account to another.
+This is a standard approach using`pg_dump` and `psql`.
 
 **The only difference between the processes is at Step 3**
```

Comment:
Now that there's only one process, this doesn't make sense

Reply:
fixed. thank you.